book/python/api.md

```diff
@@ -368,6 +372,9 @@ Return a AlphabetCounts object initialized with a count of zero for each letter
 
 ```python
 class StateBuilder(object):
+    def name(self):
+        return "AlphabetCountsStateBuilder"
```

Comment:
I think "Alphabet Counts State Builder" (and spaces on the others as well) is better.